tool-plugins/intellij/getting-started/plugin-features/README.md

```diff
@@ -74,17 +62,29 @@ You can reformat the Ballerina codes by pressing `Ctrl`+`Alt`+`L`.
 
 ![alt text](images/Formatting.gif)
 
-## Path/Query Parameter validation
+## Parameter Information
+
+You can view the required parameters of a functions, actions, etc by pressing `Ctrl`+`P`.
```

Comment:
Always add a full stop after etc. These are originally latin terms and the short form always requires full stops in the abbreviation even if it is in the middle of a sentence.

Reply:
Added. Thanks for the explanation.